src/main/resources/META-INF/common_at.cfg

```diff
@@ -247,6 +247,9 @@ public net.minecraft.tileentity.TileEntityHopper func_145896_c(I)V # setTransfer
 public net.minecraft.tileentity.TileEntitySkull field_145910_i # skullRotation
 
 protected-f net.minecraft.util.EntityDamageSourceIndirect field_76387_p # indirectEntity
+public net.minecraft.util.CooldownTracker$Cooldown
+public net.minecraft.util.CooldownTracker$Cooldown field_185137_a # Lnet/minecraft/util/CooldownTracker$Cooldown;createTicks:I
+public net.minecraft.util.CooldownTracker$Cooldown field_185138_b # Lnet/minecraft/util/CooldownTracker$Cooldown;expireTicks:I
```

Comment:
Afaict you dont use these and thus they can be removed.

Reply:
I added these to access the inner `Cooldown' class and its field 'expireTicks' as they weren't public. Is there a better way to do this? Don't really know much about this piece. 

I can remove the createTicks one though as I don't use it anymore.